### DISCUSSION TITLE
Script does not start and give me back the prompt

### DISCUSSION THREAD

**Discussion by PieTr031** [tteck/Proxmox#2162] [Q&A]
Hello all,

I have 3 ProxMox and I love using all the script furnished.

Thanks for that :)

BUT, since 3 days, one of them are giving me hard time!!!

When I tried to launch a script, the prompt give back without launching the script. (cf PJ)
![image](https://github.com/tteck/Proxmox/assets/25844959/641d5317-44e8-4ab6-b427-850d515a6d08)

You will find below the logs i've founded.
![image](https://github.com/tteck/Proxmox/assets/25844959/0879930e-d30e-41d5-b97f-26e204a97040)

Anyone have an idea?

Thanks

**Comment by tteck** [comment]
1) why are you in `/tmp` directory?
2) if nothing happens after executing the command, you're facing a network problem, _probably_ linked to DNS.

**Reply by PieTr031** [reply]
I thought i answered but it seems not sorry.

Your idea was the good one, thanks a lot :)

**Comment by PieTr031** [comment]
Hello,

Thanks for you're answer.

1. I'm in /tmp cause I tought it was a right issues.
2. Ok, that was my second idea but after reboot and ping google, I tought it was not network! I'll recheck and hold you.


Thanks :)
